mcsema/BC/Function.cpp

```diff
@@ -260,6 +289,132 @@ static llvm::BasicBlock *GetOrCreateBlock(TranslationContext &ctx,
   return block;
 }
 
+static void LiftExceptionFrameLP(TranslationContext &ctx,
+                                 const NativeFunction *cfg_func) {
+
+  if (cfg_func->eh_frame.size() > 0) {
+
+    // The personality function `__gxx_personality_v0` is lifted as global variable. Erase the
```

Comment:
I think you should do this stuff for `__gxx_personality_v0` in a different place. I.e. add this custom corner case to CFG.cpp or the stuff that initially creates the global variables.

Really, I'd like to know if this is really a problem that should be solved here, earlier in the CFG decoding, or elsewhere (e.g. idapython / binja plugins).

Reply:
Similarly, you might end up just being able to put `__gxx_personality_v0` into an ABI library.